### PR TITLE
Added more method for FFS and wallet, allow for default token.

### PR DESCRIPTION
**Pull request by leonardge**
In this PR, I have added more methods for FFS and also wallet part of the API. I have also create a method call `set_token` which allow the user to set the FFS token so in the subsequent call no token is need to. 
Some more testing related to creating the address has also been added.

### CODE REVIEW

File: tests/integration/test_ffs.py [hunk 1]
```diff
@@ -32,6 +32,40 @@ def test_grpc_ffs_add_to_hot(pygate_client: PowerGateClient, ffs_instance):
...
+    assert len(res.addrs) == 2
+    
+    expected_addr = AddrInfo(name=new_addr_name, addr=addr.addr, type='bls')
+    assert expected_addr in res.addrs
+    
+    
 def test_chunks():
     for _ in range(1):
-        yield AddToHotRequest(chunk=bytes("test_content", "ASCII"))
+        yield AddToHotRequest(chunk=bytes('test_content', 'ASCII'))
```
Comment (apogiatzis) [inline tests/integration/test_ffs.py:71]: Looking good man!! Just another minor thing, it seems that your IDE converts double quotes to single quotes, can you run `pipenv run format` before you push just to keep the quoting style consistent?

Comment (leonardge) [reply tests/integration/test_ffs.py]: This is a handy command! The formatting issue should be fixed now.

**Comment by leonardge** [comment]
I think we should add type annotation next time. The code has been pretty messy and hard to read now.

**Comment by apogiatzis** [comment]
> I think we should add type annotation next time. The code has been pretty messy and hard to read now.

Yup type annotation would be a good addition

**Pull request merged**